components/k8s-model-server/images/releaser/components/workflows.libsonnet

```diff
@@ -139,6 +224,100 @@
                 },
               ],
             },  // build-tf-serving-image
+            {
+              name: "deploy-tf-serving",
+              container: {
+                command: [
+                  "python",
+                  "-m",
+                  "testing.test_deploy",
+                  // "--project=mlkube-testing",
+                  // "--cluster=kubeflow-testing",
+                  // "--zone=us-east1-d",
+                  "--project=kai-test2",
+                  "--cluster=kai-kubeflow-testing",
+                  "--zone=us-central1-a",
+                  "--github_token=$(GIT_TOKEN)",
+                  "--test_dir=" + testDir,
+                  "--artifacts_dir=" + artifactsDir,
+                  "--deploy_core=False",
+                  "--deploy_tf_serving=true",
+                  "--model_server_image=" + serving_image,
+                  "--test_inception=true",
+                  // TODO: use kubeflow image
+                  "--inception_client_image=gcr.io/kai-test2/incpetion-client:1.0",
+                ],
+                env: [
```

Comment:
Same comment as above. Lets create a template or use the build_template to set common environment variables on all steps.